dev_apps/music-nga/views/albums/view.js

```diff
@@ -56,6 +57,14 @@ AlbumsView.prototype.getAlbums = function() {
     .then(albums => clean(albums));
 };
 
+AlbumsView.prototype.search = function(query) {
+  return this.fetch('/api/search/album/' + query).then((response) => {
```

Comment:
um.. disregard that last comment.. i clearly need more coffee today.. i thought this call was directly a part of the event listener above *facepalm*